lib/functions.php

```diff
@@ -106,6 +109,10 @@ function digest_site(ElggUser $user, $interval) {
 	unset($userdata);
 	
 	// restore current user
+	if ($is_impersonate_session) {
+		logout();
```

Comment:
we can't use the function `logout()` as there are events fired when you call this function. we don't want that to happen